lib/charms/postgresql_k8s/v0/postgresql.py

```diff
@@ -604,12 +604,11 @@ def build_postgresql_parameters(
                 # Use 25% of the available memory for shared_buffers.
                 # and the remaining as cache memory.
                 shared_buffers = int(available_memory * 0.25)
+                parameters["shared_buffers"] = f"{int(shared_buffers * 128 / 10**6)}"
             effective_cache_size = int(available_memory - shared_buffers)
-            parameters.setdefault("shared_buffers", f"{int(shared_buffers / 10**6)}MB")
-            parameters.update({"effective_cache_size": f"{int(effective_cache_size / 10**6)}MB"})
-        else:
-            # Return default
-            parameters.setdefault("shared_buffers", "128MB")
```

Comment:
I removed this `setdefault` of 128MB because that's what postgresql have as a default normally? No need to set it like this AFAIK, LMK if I'm missing something

Reply:
You're right. I tested with the `testing` profile, and it worked correctly, even after changing the configuration and later resetting it.